src/components/Card/Card.tsx

```diff
@@ -6,11 +6,11 @@ import { useAppDispatch } from 'store/hooks';
 import { useDrag, useDrop } from 'react-dnd';
 import { moveCard, removeCard, renameCard } from 'slices/columnSlice';
 import { updateTextAreaHeight } from 'helpers/functions/updateTextAreaHeight';
-import { ModalRenameCard } from 'components/ModalRenameCard/ModalRenameCard';
 import { formatDate } from 'helpers/functions/formatDate';
 import { getCurrentColor } from 'helpers/functions/getCurrentColor';
 import { useOpener } from 'hooks/useOpener';
-import { CardModal } from 'components/CardModal/CardModal';
+import { CardModal } from 'components/CardModal';
+import { ModalRenameCard } from 'components/RenameCardModal';
```

Comment:
you renamed component folder name, but not component name. These two names should be same